### PR TITLE
[merged] doc: Added CPD in development standards.

### PR DESCRIPTION
**Pull request by ashcrow**
Adds a blurb in the development doc about the CPD process for large changes (https://github.com/projectatomic/commissaire/wiki#commissaire-proposal-documents)

### CODE REVIEW

File: doc/development.rst [hunk 1]
```diff
@@ -43,6 +43,13 @@ Like most projects commissaire expects specific coding standards to be followed.
 `pep8 <https://www.python.org/dev/peps/pep-0008/>`_ is followed strictly with
 the exception of E402: module level import not at top of file.
 
+Commissaire Proposal Document
+`````````````````````````````
+A Commissaire Proposal Document (CPD) is needed when a change will significantly
+change the current implementation. This may mean breaking backward
```
Comment (mbarnes) [inline doc/development.rst]: Perhaps better wording, I'll let you decide.
`... must be submitted and agreed upon before significantly changing the current implementation.  This applies to ...`

Comment (mbarnes) [inline doc/development.rst]: Perhaps better wording, I'll let you decide.
`... must be submitted and approved before significantly changing the current implementation.  This applies to ...`

**Comment by mbarnes** [comment]
Weird, my inline comment keeps disappearing.  Was trying to suggest alternate wording.

`A ... (CPD) must be submitted and approved before significantly changing the current implementation.  This applies to ...`

**Comment by ashcrow** [comment]
Will update shortly.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by mbarnes** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**